force-app/main/default/lwc/attendanceRow/attendanceRow.js

```diff
@@ -102,4 +111,21 @@ export default class AttendanceRow extends LightningElement {
         this.localRecord[this.fields.quantity.fieldApiName] = event.detail.value;
         this._isEdited = true;
     }
+
+    get showSkip() {
+        return !this.recordId && !this.readOnly;
+    }
+
+    handleToggleButton() {
+        this.rowDisabled = !this.rowDisabled;
+        if (this.rowDisabled) {
+            let inputFields = [
+                ...this.template.querySelectorAll("lightning-input"),
+                ...this.template.querySelectorAll("lightning-input-field"),
+            ];
+            inputFields.forEach(field => {
+                field.value = null;
```

Comment:
One question here. Do we need to set the value to null here when we disable a row.

Reply:
Yes, this was a late-breaking AC from Meg yesterday. it makes the UX much clearer what's happening. I think I added it to the ACs.